### ISSUE TITLE
[Bug] [a11y] Some options from the appearance menu are described as "not checked" when are checked

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3476]
## Steps to reproduce
1. Turn Talkback on.
1. Go to a Wikipedia article and activate Reader mode.
1. Tap on Appearance and swipe through all options.

### Expected behavior
When an option is highlighted, it is announced: "Checked Light color scheme radio toggle".

### Actual behavior
Options Sans serif, Dark, Sepia are announced: "Not checked...", even though they are highlighted.

### Device information
* Android device: HTC 10 (Android 8), Sony Xperia Z5 Premium (Android 7.1.1)
* Fenix version: 1.0.1924 (#11650657)

**Comment by mcarare** [comment]
@softvision-miralobontiu I cannot reproduce this on my devices. Can you test it again, please?

**Comment by softvision-miralobontiu** [comment]
I was not able to reproduce this issue anymore.
Tested on RC v2.2.0-rc.2 and Nightly 10/17 build, with Huawei Honor 8 (Android 7), and Google Pixel (Android Q).
I will close this issue.
